### DISCUSSION TITLE
Is it possible to prevent specific pages from showing up in archive pages?

### DISCUSSION THREAD

**Discussion by Chow-Ray** [adityatelange/hugo-PaperMod#1454] [Q&A]
I have a page that I don't want to show on the archive page, is there a page setting like this?

For example page https://chowray.netlify.app/sponsor/ and page https://chowray.netlify.app/archives/
![2](https://github.com/adityatelange/hugo-PaperMod/assets/29678677/760ef426-ce4d-4546-841f-031d902e1efb)

**Comment by RoneoOrg** [comment]
See https://github.com/adityatelange/hugo-PaperMod/issues/1110

**Reply by Chow-Ray** [reply]
I added code `hiddenInHomeList: true` to the `post.md`, but it did not take effect. Do I need to add any code to file `config.yml`?
```
---
title: "Sponsor 赞助者"
date: 2024-03-07T20:23:04+08:00
draft: false
ShowWordCount: false
ShowReadingTime: false
comments: false
hideFooter: false
ShowBreadCrumbs: false
disableShare: false
hideMeta: true
showToC: false
hiddenInHomeList: true
---
## How to Sponsor 如何赞助
```

**Reply by Chow-Ray** [reply]
@RoneoOrg @adityatelange Please help me, thank you.
